lighthouse-core/report/html/renderer/util.js

```diff
@@ -556,10 +555,11 @@ class Util {
    * @param {LH.Locale} locale
    */
   static setNumberDateLocale(locale) {
-    Util.numberDateLocale = locale;
-
     // When testing, use a locale with more exciting numeric formatting
-    if (Util.numberDateLocale === 'en-XA') Util.numberDateLocale = 'de';
+    if (locale === 'en-XA') locale = 'de';
+
+    Util.numberDateLocale = locale;
+    Util.numberFormatter = new Intl.NumberFormat(locale);
```

Comment:
more static-ish state is unfortunate, but it seemed wasteful to do `new Intl.NumberFormat(Util.numberDateLocale).format(coarseTime)` for each format call...I'm not sure if it matters much

(tsc ensures it's initialized on the class, though, so that's a plus)